### ISSUE TITLE
ruby-unicorn version mismatch between Arch community and quarry

### ISSUE THREAD

**Issue by Zerophase** [anatol/quarry#73]
In the community repo ruby-unicorn is at version 1.0, and on here it's version 5.2.0. Looking further into the Arch package that version of ruby-unicorn seems to be a different project related to qemu. Maybe, ruby-unicorn in quarry should have a different name, so as not to conflict with the Arch package?

**Comment by anatol** [comment]
@anthraxx

Unicorn is a widely known project in ruby community. It is unfortunate that we have another project that has a collision name with `ruby-unicorn` from quarry.

**Comment by anthraxx** [comment]
Yeah thats really a pity, but the other one is not even packaged for AUR outside of your cool custom repo :yum:

**Comment by Zerophase** [comment]
Ideally, I think the one in community should be ruby-unicorn-engine.

**Comment by anthraxx** [comment]
that would violate the naming that is choosen right now. Well i'm really sorry but right now i have hard time considering to rename it... as it didn't even seem "important enough" to get packaged directly in the first place.

**Comment by anatol** [comment]
Another option is to call the new package `unicorn-engine`. The upstream uses the name for their domain name http://www.unicorn-engine.org/ and github repo https://github.com/unicorn-engine

**Comment by Zerophase** [comment]
unicorn engine, would need that name to get the ruby code into the list of ruby gems too.

**Comment by anthraxx** [comment]
"unicorn" is what the gem, all other bindings and the project itself is named. the space 'unicorn' was already taken and it belongs to a family of tools (capstone, keystone) that share the same naming for the core projects itself, bindings and gem names etc.

**Comment by Zerophase** [comment]
I just don't see the ruby gem tie in listed on RubyGems.org. (I'm assuming the ruby code is a gem) https://rubygems.org/search?page=3&query=unicorn&utf8=%E2%9C%93

**Comment by anthraxx** [comment]
I was more referring to the projects source and not rubygems as a central storage, sorry for confusion

**Comment by anatol** [comment]
Taking into account that this security framework plans to distribute their ruby binding as gems they obviously cannot use `unicorn` as a name for their gem. And they need to find another name for their ruby project.
